frontend/src/components/Login/TownSelectionPart2.test.tsx

```diff
@@ -113,6 +113,8 @@ function wrappedTownSelection() {
           finishGame: () => {},
           showInstructions: false,
           showLeaderboard: false,
+          toggleShowLeaderboard: () => {},
+          gameStarted: false,
```

Comment:
gameStarted shouldn't be necessary anymore.  You might have to manually go in and delete the occurrences.

Reply:
ah nice catch! merge conflicts got me confused